setup.py

```diff
@@ -55,7 +55,6 @@ def find_version(*file_paths):
         'flask>=0.10.0',
         'lxml>=3.4.4',
         'spyne>=2.12.8',
-        'nose',
```

Comment:
Nose is not used. Ideally we should read in from the requirements file here too to prevent the dependencies from keeping out of sync. The main issue to do that is the lack of a PyPI `odm2api` package, since `geoalchemy` does not seem to be used at all.